python/build-wheel-manylinux2014.sh

```diff
@@ -129,7 +137,9 @@ done
 # Clean the build output so later operations is on a clean directory.
 git clean -f -f -x -d -e .whl -e python/ray/dashboard/client
 
-echo "--- Build JAR"
-if [ "${BUILD_JAR-}" == "1" ]; then
-  ./java/build-jar-multiplatform.sh linux
+if [[ "${RAY_INSTALL_JAVA}" == "1" ]]; then
+  if [[ "${BUILD_JAR-}" == "1" ]]; then
```

Comment:
Combine into one if?

Reply:
`BUILD_JAR` is only used in linux wheels and jars build. seems that not every thing that builds java package it into a jar.